src/components/ComparateurPublicodes/ConfigurationDropdown.tsx

```diff
@@ -0,0 +1,253 @@
+import { useEffect, useRef, useState } from 'react';
+
+import Button from '@/components/ui/Button';
+import Icon from '@/components/ui/Icon';
+import Loader from '@/components/ui/Loader';
+import useCrud from '@/hooks/useCrud';
+import { type ProComparateurConfigurationResponse } from '@/pages/api/pro/comparateur/configurations/[[...slug]]';
+import { notify } from '@/services/notification';
+import cx from '@/utils/cx';
+import { sortKeys } from '@/utils/objects';
+
+const ConfigurationDropdown = ({
+  configuration,
+  onChangeConfiguration,
+  loadWhenOnlyOneConfig,
+}: {
+  configuration: Record<string, any>;
+  onChangeConfiguration: (configuration: Record<string, any>) => void;
+  loadWhenOnlyOneConfig: boolean;
+}) => {
+  const [loaded, setLoaded] = useState<boolean>(false);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const [isAddingNew, setIsAddingNew] = useState<boolean>(false);
+  const [newConfigName, setNewConfigName] = useState<string>('');
+  const [renamingConfig, setRenamingConfig] = useState<string | null>(null);
+  const [renameValue, setRenameValue] = useState<string>('');
+  const dropdownRef = useRef<HTMLDivElement>(null);
+  const { address, ...situation } = configuration;
```

Comment:
Ici y'a un warning eslint qui dit que address n'est pas utilisé.
```suggestion
  const { address: _, ...situation } = configuration;
```